docs/Tutorials/Contracts/Calling-Contract-Functions.md

```diff
@@ -42,7 +42,7 @@ contract SimpleStorage {
 Once the contract is deployed, you can perform a read operation using the `get` function call and a
 write operation using the `set` function call.
 This tutorial uses the [web3js](https://www.npmjs.com/package/web3) library to interact with the contract.
-A [full example](https://github.com/ConsenSys/quorum-dev-quickstart/blob/master/files/besu/smart_contracts/scripts/public_tx.js)
+A [full example](https://github.com/ConsenSys/quorum-dev-quickstart/blob/b8f7e3bc1d5f773b823c5e9983e9da11bb552ed7/files/besu/smart_contracts/privacy/scripts/public_tx.js)
```

Comment:
Check [1e8cc2](https://github.com/ConsenSys/quorum-dev-quickstart/commit/1e8cc281098923802845cd829ec20c88513c2e1c) for the lastest version of the file, modified a few hours ago.